Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.py

```diff
@@ -0,0 +1,784 @@
+                }
+            },
+            "alert": {
+                "data": {
+                    "id": "101", "type": "alert"
+                    "id": "101", "type": "alert"
+                }
+            }
+        }
+        Args:
+            kwargs (dict): Dict containing key values parameters to be used for relationships. for example:
+            {'ip_version': 4}
+
+        Returns:
+            (dict): Netscout relationships object
+        """
+        relationships: Dict[str, Any] = {}
+        for key, val in kwargs.items():
+            if val:
+                # In some cases the name of the relationships is not the same as the type (most cases it is)
+                _type = self.RELATIONSHIP_TO_TYPE.get(key, key)
+                if key == 'routers':
+                    relationships[key] = {
+                        'data': [{
+                            'type': _type,
+                            'id': val[0]
+                        }]
+                    }
+                else:
+                    relationships[key] = {
+                        'data': {
+                            'type': _type,
+                            'id': val
+                        }
+                    }
+        return relationships
+
+    def build_data_attribute_filter(self, attributes_dict: dict) -> str:
+        """
+        Builds data attribute filter in the NetscoutArbor form. For example: '/data/attributes/importance>1' where
+        key=importance operator='>' and value=1.
+        The function iterates over all arguments (besides operators listed in the OPERATOR_NAME_DICTIONARY) and chain
+        together the 'key operator val' such that the argument name is 'key', its value is 'val' and operator is '=' if
+        no relevant operator is present. In case of multiple parameters the attributes are separated with 'AND'.
+
+        Args:
+            attributes_dict (dict): Dict containing key values filter parameters. for example: {'importance': 1}
+
+        Returns:
+            (str): Netscout data attribute filter string. For example:
+            /data/attributes/importance>1 AND /data/attributes/ongoing=true
+        """
+        param_list = []
+        operator_names = self.OPERATOR_NAME_DICTIONARY.values()
+        for key, val in attributes_dict.items():
+
+            # We don't create a filter for operator names
+            if key not in operator_names and val:
+                operator = '='  # type: str
+
+                # If the current parameter supports a special operator (it appears in the OPERATOR_NAME_DICTIONARY),
+                # we take the operator value using the operator name (that appears in the OPERATOR_NAME_DICTIONARY)
+                if operator_name := self.OPERATOR_NAME_DICTIONARY.get(key):
+                    operator = attributes_dict.get(operator_name) if attributes_dict.get(
+                        operator_name) else '='  # type: ignore
```

Comment:
if we cant get rid of those type ignores, lets at least set the error code and not ignore all